### PR TITLE
Enable spellchecking for Rust files

### PR DESCRIPTION
**Pull request by Turbo87**
This only works for comments as of now, but can be extended to work for strings and identifiers as well.

bildschirmfoto 2015-11-06 um 16 23 41

### CODE REVIEW

**Comment by Turbo87** [comment]
```
Server returned HTTP response code: 503 for URL: https://services.gradle.org/distributions/gradle-2.8-bin.zip
```

.... tests run fine though

File: src/org/rust/lang/spellchecker/LiteralExpressionTokenizer.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import com.intellij.spellchecker.tokenizer.EscapeSequenceTokenizer
+import com.intellij.spellchecker.tokenizer.TokenConsumer
+import org.rust.lang.core.psi.RustLitExpr
+
+class LiteralExpressionTokenizer : EscapeSequenceTokenizer<RustLitExpr>() {
+
+    override fun tokenize(element: RustLitExpr, consumer: TokenConsumer) {
+        val text = element.stringLiteral?.text ?: return
+
+        if (!text.contains("\\")) {
```
Comment (matklad) [inline src/org/rust/lang/spellchecker/LiteralExpressionTokenizer.kt]: `if ("\\" !in text)`

Comment (pavel-v-chernykh) [inline src/org/rust/lang/spellchecker/LiteralExpressionTokenizer.kt]: Yeah, I have been caught on that recently :smile:

**Comment by Turbo87** [comment]
@matklad done

File: src/org/rust/lang/spellchecker/LiteralExpressionTokenizer.kt (hunk 1 above)
Comment (Turbo87) [inline src/org/rust/lang/spellchecker/LiteralExpressionTokenizer.kt]: done

**Pull request merged**